<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buslineWidget</class>
 <widget class="QWidget" name="buslineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#buslineWidget {
border-image: url(:/images/background.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>380</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame {border-image: url(:/images/backglass.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>线路：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>票价：</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/check.png);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDlabel_13">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;
image: url(:/images/uncheck.png);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>15</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>设 置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2 {
border-image: url(:/images/background.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
